packages/angular/build/src/builders/karma/schema.json

```diff
@@ -229,7 +229,8 @@
     },
     "watch": {
       "type": "boolean",
-      "description": "Run build when files change."
+      "description": "Re-run tests when source files change.",
```

Comment:
Double-checking: Was it ever a thing that the karma builder would default to non-watching outside of TTY?